book/src/config/api.md

```diff
@@ -14,7 +14,6 @@ target_path = "."
 # auto_path = "src/auto"
 work_mode = "normal"
 # Whether the library uses https://gitlab.gnome.org/GNOME/gi-docgen for its documentation
-use_gi_docgen = false
```

Comment:
It's going to take a while until everything uses this so for the time being maybe change `use_gi_docgen` to `doc_format` and allow the same values there, and don't make the attribute in the XML mandatory yet?

Reply:
What is everything? It is GTK, GLib, Pango and libadwaita (merged already https://gitlab.gnome.org/GNOME/libadwaita/-/merge_requests/1428).

The remaining bindings don't get updated often at all. 

If you just want to ignore the element for now, it could be done in a separate patch by someone else.

Reply:
If everything using gi-docgen is updated then that's fine with me. Just default to "unknown" if the attribute is not in the XML.